working/augmentation-libraries/feature-specification.md

```diff
@@ -719,6 +716,12 @@ It is a **compile-time error** if:
     const variable be augmented by another const variable, changing its value,
     or is that too weird?)**
 
+*   An `abstract` variable is augmented with a non-abstract variable, getter,
+    or setter.
```

Comment:
Github doesn't allow to comment lines 679-683.

> *   An augmenting declaration uses `augmented` when the augmented declaration
>     has no concrete implementation. Note that all external declarations are
>     assumed to have an implementation provided by another external source, and
>     otherwise they will throw a runtime error when called. An `abstract`
>     variable introduces no implementation.

Probably the last sentence here can be removed. If augmenting an abstract variable by non-abstract variable, getter or setter is an error itself we don't care about `augmented` expression into these augmentations any longer.

Reply:
Agreed that seemed unnecessary, removed